components/icons/Icons.tsx

```
@@ -126,3 +128,92 @@ export const IconDirectory = () => {
+export const IconEverworks = (props?: SVGProps<SVGSVGElement>) => {
+  const { className, ...rest } = props || {};
+  return (
+    <svg
+      {...rest}
+      // height="200"
+      viewBox="0 0 512 512"
+      fill="none"
+      fontWeight="700"
+      xmlns="http://www.w3.org/2000/svg"
+      className={className}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)